packages/commonwealth/client/scripts/state.ts

```diff
@@ -97,6 +97,8 @@ export interface IApp {
   sidebarToggled: boolean;
 
   loginState: LoginState;
+  loginStateEmitter: EventEmitter;
```

Comment:
React wasn't tracking updates to app.loginState, so I added this emitter to listen for. I don't _love_ this solution, but once we address state management, this is one area to address.

Reply:
I think this is a great interim solution, and definitely top of list for state management.